velox/common/memory/tests/MockSharedArbitratorTest.cpp

```diff
@@ -3935,5 +3933,90 @@ TEST_F(MockSharedArbitrationTest, concurrentArbitrationWithTransientRoots) {
   }
   controlThread.join();
 }
-} // namespace
+
+TEST_F(MockSharedArbitrationTest, arbitrationOperationTimedLock) {
```

Comment:
This test should be in ArbitrationParticipantTest? Also cover the two APIs: abort and reclaim?

In MockSharedArbirrationTest, let's have two test cases with one for reclaim lock timeout and one for abort lock time out? Thanks!